spec/build/rake_tasks_spec.rb

```diff
@@ -210,16 +210,16 @@ def releases_response(*versions)
       (top + 'public/version-aliases/go.json').read
     )
     expect(aliases).to eq(
-      '1' => '1.9.1',
+      '1' => '1.9.3',
       '1.2' => '1.2.2',
       '1.2.3' => '1.2.3',
       '1.2.x' => '1.2.3',
       '1.4.0' => '1.4.0',
       '1.4.x' => '1.4.0',
-      '1.9.1' => '1.9.1',
-      '1.9.x' => '1.9.1',
-      '1.x' => '1.9.1',
-      '1.x.x' => '1.9.1',
+      '1.9.3' => '1.9.3',
+      '1.9.x' => '1.9.3',
+      '1.x' => '1.9.3',
+      '1.x.x' => '1.9.3',
```

Comment:
The spec that is failing in this file is due to these values having been changed.  The expectation here is driven by stubbed test data and _should not_ be considered a part of the production code go version aliases 💖 

Said another way: revert the changes to this file and the specs should pass.